tests/integration/db.go

```diff
@@ -100,7 +100,7 @@ func NewBadgerFileDB(ctx context.Context, t testing.TB) (client.DB, error) {
 // setupDatabase returns the database implementation for the current
 // testing state. The database type on the test state is used to
 // select the datastore implementation to use.
-func setupDatabase(s *state) (client.DB, string, error) {
+func setupDatabase(s *state) (*node.Node, string, error) {
```

Comment:
suggestion: maybe we should call it `setupNode`?

Reply:
I agree. Will do.

- [ ] rename `setupDatabase` to `setupNode`